src/components/Display.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const divStyle = {
+  textAlign: 'right',
+};
+
+function Display(props) {
+  const { result } = props;
+  return (
+    <form action="">
+      <input type="text" id="lname" name="lname" style={divStyle} className="form-control Calc" value={result} disabled />
```

Comment:
- [ ] [MANDATORY] It is not best practice to use inline CSS. Read more about it in this [Article](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846).